### PR TITLE
add-docs-chalice

### PR DESCRIPTION
**Pull request by Gleekzone**
adding docs for ChaliceIntegration

### CODE REVIEW

File: src/docs/platforms/python/chalice.mdx [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+draft: false
+categories: []
+toc: true
+title: Chalice
+sidebar_order: 2
+tags: []
+---
+The Chalice integration adds support for the [Chalice aws](https://aws.github.io/chalice/).
```
Comment (untitaker) [inline src/docs/platforms/python/chalice.mdx:9]: Please use the same formatting as in https://github.com/getsentry/sentry-docs/blob/master/src/docs/platforms/python/pyramid.mdx

1. "new in version" phrasing and italics (and in parenthesis)

**Review by untitaker — APPROVED** [review]

File: src/docs/platforms/python/chalice.mdx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+The integration is avalible since 1.16.0
```
Comment (untitaker) [inline src/docs/platforms/python/chalice.mdx:10]: Wrong version btw, let's do 0.17

Comment (Gleekzone) [reply src/docs/platforms/python/chalice.mdx:10]: I mean the version 1.16.0 of chalice. What version 0.17 is related to?

Comment (untitaker) [reply src/docs/platforms/python/chalice.mdx]: SDK version, adding minimum supported Chalice version is also good to have

**Comment by untitaker** [comment]
Continuing at https://github.com/getsentry/sentry-docs/pull/2220

**Pull request closed without merging**